src/main/java/reauth/Main.java

```diff
@@ -12,7 +12,7 @@
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 
-@Mod(modid = "ReAuth", guiFactory = "reauth.GuiFactory", canBeDeactivated = true, clientSideOnly = true, acceptedMinecraftVersions = "[1.8.0,1.8.9]")
+@Mod(modid = "ReAuth", guiFactory = "reauth.GuiFactory", canBeDeactivated = true, clientSideOnly = true, acceptedMinecraftVersions = "[1.9.0,1.9.4]")
```

Comment:
Forgot to change the modid to lower case as server and client logs all say that modid's need to be lower case.